Bender.yml

```diff
@@ -5,31 +5,33 @@ package:
 
 dependencies:
-  axi2mem:              { git: "git@github.com:pulp-platform/axi2mem.git", rev: "6973e0434d26ba578cdb4aa69c26c1facd1a3f15" } # deprecated, replace with axi_to_mem in axi repo
-  axi2per:              { git: "git@github.com:pulp-platform/axi2per.git", version: 1.0.1 }
-  per2axi:              { git: "git@github.com:pulp-platform/per2axi.git", version: 1.0.4 }
-  axi_size_conv:        { git: "git@github.com:pulp-platform/axi_size_conv.git", rev: "5239f87fe772111ec368fb08dbb971516edce097" } # deprecated, replace with axi_dw_converter in axi repo
-  cluster_interconnect: { git: "git@github.com:pulp-platform/cluster_interconnect.git", version: 1.1.0 }
-  event_unit_flex:      { git: "git@github.com:pulp-platform/event_unit_flex.git", rev: "1.4.1" }
-  mchan:                { git: "git@github.com:pulp-platform/mchan.git", version: 1.2.2 }
-  hier-icache:          { git: "git@github.com:pulp-platform/hier-icache.git", version: 1.2.0 }
-  icache_mp_128_pf:     { git: "git@github.com:pulp-platform/icache_mp_128_pf.git", rev: "6f2e54102001230db9c82432bf9e011842419a48" }
-  icache_private:       { git: "git@github.com:pulp-platform/icache_private.git", rev: "1d4cdbcbec3ab454c09d378fc55631b60450fccd" }
-  cluster_peripherals:  { git: "git@github.com:pulp-platform/cluster_peripherals.git", version: 2.1.0 }
-  fpu_interco:          { git: "git@github.com:pulp-platform/fpu_interco.git", rev: "83c7a1b690bffd006d8b925805aab3c1f8ba9ff1" }
-  axi:                  { git: "git@github.com:pulp-platform/axi.git", version: 0.7.1 } # to be updated (now 0.24?)
-  axi_node:             { git: "git@github.com:pulp-platform/axi_node.git", version: 1.1.4 } # deprecated, replaced by axi_xbar (in axi repo)
-  axi_slice:            { git: "git@github.com:pulp-platform/axi_slice.git", version: 1.1.4 } # deprecated, replaced by axi_cut (in axi repo)
-  axi_slice_dc:         { git: "git@github.com:pulp-platform/axi_slice_dc.git", version: 1.1.3 } # deprecated, replaced by axi_cdc (in axi repo)
-  timer_unit:           { git: "git@github.com:pulp-platform/timer_unit.git", version: 1.0.2 }
-  common_cells:         { git: "git@github.com:pulp-platform/common_cells.git", version: 1.13.1 }
-  tech_cells_generic:   { git: "git@github.com:pulp-platform/tech_cells_generic.git", version: 0.2.2 }
-  riscv:                { git: "git@github.com:micprog/cv32e40p.git", rev: "pulpissimo-v3.4.1"} # To be updated to openhwgroup repository
-  ibex:                 { git: "git@github.com:lowRISC/ibex.git", rev: "b148f3af51066ac3537c7c6eb4acbc17004bf868" }
-  scm:                  { git: "git@github.com:pulp-platform/scm.git", version: 1.0.1}
-  hwpe-datamover-example: { git: "https://github.com:pulp-platform/hwpe-datamover-example.git", version: 1.0 }
+  axi2mem:                { git: "https://github.com/pulp-platform/axi2mem.git", rev: "6973e0434d26ba578cdb4aa69c26c1facd1a3f15" } # deprecated, replace with axi_to_mem in axi repo
```

Comment:
quick question: did you change from git to https on purpose? If yes, you may want to state this somewhere.

Reply:
This change was on purpose, as cloning `https://` links is possible without a github account, while `git@` requires you to have an ssh key stored. I was unaware of this when I started out with updates, but I started fixing this once it was pointed out to me.